### ISSUE TITLE
Error in victim side

### ISSUE THREAD

**Issue by aviran86** [GreatSCT/GreatSCT#5]
Hi
someone know what the problem ?
![12](https://user-images.githubusercontent.com/7000498/40665876-0b84fdfe-6367-11e8-86f7-cd9032bc324d.JPG)

**Comment by ConsciousHacker** [comment]
Yes, all the assembly payloads are 32 bit. Use the 32 bit version of RegAsm.
